lib/importer/import-default.js

```diff
@@ -0,0 +1,287 @@
+"use strict";
+
+const stepStates = require('./import-step-states'),
+        execSync = require('child_process').execSync,
+         rfc6902 = require('rfc6902'),
+            path = require('path'),
+              fs = require('fs-extra');
+
+/**
+ * Processes metadata located in <pkg-name@version?.json> metadata file
+ *
+ * Steps in execution order
+ * 0. register
+ *
+ * 1. applyPatches
+ * 2. registerDependencies
+ * 3. registerBundles
+ * 4. saveProject
+ * 5. installTasks
+ * 6. executeScripts
+ */
+module.exports = class {
+
+    constructor() { }
+
+    /**
+     * Register this class with ImportEngine
+     *
+     * @param engine ImportEngine ImportEngine instance
+     * @param options any given CLI arguments
+     * @param projectInfo basic information about the project itself
+     *
+     */
+    register(engine, options, projectInfo) {
+        this.engine = engine;
+        this.cliParams = options;
+        this.projectInfo = projectInfo;
+        this.ui = this.engine.cliBridge.ui;
+
+        this.bundles = this.engine.projectConfig.build.bundles;
+        if (this.bundles.length === 0) {
+            throw new Error("aurelia.json: bundles section is missing.");
+        }
+
+        this.bundleName = this.cliParams.bundle || 'vendor-bundle.js';
```

Comment:
I think what we'll want to do here is search for the bundle with the largest number of dependencies and make that the fallback bundle.